### ISSUE TITLE
Game Concept

### ISSUE THREAD

**Issue by bradleyjay** [Failure-Enthusiasts/storage_wars#12]
tl;dr: We don't know what the game is yet. So, what should it be?

Main questions here: what's the player do while they play, and what's their goal in the game?

General ideas on what we could do:

- could be battleship, but with points. You spend your points on each "attack", without knowing how many sinks the defender, goal is to finish the game with some points left
- maybe it's a grid, where we map container addresses to location. Player interacts with the 5x5 grid locations under the hood we use addresses
- maybe "attacks" are scheduled in some way, so you're not manually pushing go each time, keep things moving
- maybe we're DDOSing the defending container. Maybe it's giving them files to fill up their RAM, giving them something to do that overloads their CPU, etc etc....

-----

For now? Literally battleship. 

Later? Who knows.
